src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/multinomial.cpp

```diff
@@ -0,0 +1,145 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/multinomial.hpp"
+
+#include <openvino/core/type/element_type.hpp>
+#include <openvino/runtime/tensor.hpp>
+
+using namespace CPUTestUtils;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+namespace Multinomial {
+
+const int GLOBAL_SEED = 1;
+const int OP_SEED = 2;
+
+const std::vector<float> probs_4x4_f32 = {0.00001f,
+                                          0.001f,
+                                          0.1f,
+                                          10.0f,
+                                          0.001f,
+                                          0.00001f,
+                                          10.0f,
+                                          0.1f,
+                                          0.1f,
+                                          10.0f,
+                                          0.00001f,
+                                          0.001f,
+                                          10.0f,
+                                          0.1f,
+                                          0.001f,
+                                          0.00001f};
+
+const std::vector<ov::float16> probs_2x3_f16 = {(ov::float16)0.001f,
+                                                (ov::float16)0.1f,
+                                                (ov::float16)10.0f,
+                                                (ov::float16)10.0f,
+                                                (ov::float16)0.001f,
+                                                (ov::float16)0.1f};
+
+const std::vector<ov::bfloat16> probs_1x3_bf16 = {(ov::bfloat16)0.001f, (ov::bfloat16)0.1f, (ov::bfloat16)10.0f};
+
+const std::vector<float> probs_4x4_f32_log =
+    {3.0f, 6.0f, 10.0f, 0.0f, 3.0f, 0.0f, 10.0f, 6.0f, 6.0f, 10.0f, 0.0f, 3.0f, 10.0f, 6.0f, 3.0f, 0.0f};
+
+const std::vector<ov::float16> probs_2x3_f16_log = {(ov::float16)3.0f,
```

Comment:
```suggestion
const std::vector<ov::float16> probs_2x3_f16_log = {static_cast<ov::float16>(3.0f),
```
use C++ casting or maybe there is way to write in different way without cast (to avoid compiler warnings)

Reply:
Used class constructor with float instead